.travis.yml

```diff
@@ -58,5 +58,5 @@ install:
 script: sphinx-build -nW -b html -d docs/_build/doctrees/ docs/ docs/_build/html
 script: export OMP_NUM_THREADS=1 && pytest src/openfermion --cov src/openfermion
 
-after_success:
-  - coveralls
+# after_success:
+#   - coveralls
```

Comment:
Oh I didn't mean to actually disable the check. I meant that we could merge this PR in spite of the failing check.

Reply:
I didn't know that was possible. I thought you had to disable, merge then re-enable. I've reverted this.